src/patternfly/components/TextInputGroup/examples/TextInputGroup.md

```diff
@@ -17,6 +17,17 @@ cssPrefix: pf-c-text-input-group
 {{/text-input-group}}
 ```
 
+### Disabled
+```hbs
+{{#> text-input-group text-input-group--id="disabled" text-input-group--IsDisabled="true"}}
```

Comment:
We will typically add a little context to the ID to avoid any potential conflicts if this is used as an `id` somewhere, since an `id` can only be used once on a page.

```suggestion
{{#> text-input-group text-input-group--id="text-input-group-disabled" text-input-group--IsDisabled="true"}}
```